tests/test-files/test-multi-stage.yaml

```diff
@@ -0,0 +1,11 @@
+steps:
+  test:
+    build:
+      no-cache: true
+      dockerfile: |
+        FROM {{ DOCKER_REGISTRY }}/busybox:latest AS stage1
+        RUN echo "test" > /test-file.txt
+        FROM {{ DOCKER_REGISTRY }}/busybox:latest AS stage2
+      target: stage1
```

Comment:
This seems an insufficient test to me, what if you write another file in `stage2` and then use the something like this for the run:

```
run:
  cmds:
  - test -f /test-file1.txt
  - ! test -f /test-file2.txt
```

Reply:
and obviously change the name of the file written in `stage1` for my suggestion